src/_sass/themes/light/buttons.scss

```diff
@@ -41,3 +41,14 @@
     background-color: $white-bis;
   }
 }
+
+.button.is-rss {
+  background-color: darkorange;
+  border-color: darkorange;
+  color: white;
+
+  &:hover, &:active {
+    border-color: rgba(255, 140, 0, 0.966);
+    background-color: rgba(255, 140, 0, 0.966);
+  }
+}
```

Comment:
You've removed a newline at the end of this file here

Reply:
What is the point of having a trailing newline?

Reply:
It makes the git diffs cleaner if you maintain it